8Knot/pages/chaoss/visualizations/contrib_importance_pie.py

```diff
@@ -124,48 +153,35 @@
                                             step=1,
                                             value=10,
                                             size="sm",
+                                            style={"width": "80px"},
+                                            className="dark-input",
                                         ),
                                     ],
-                                    className="me-2",
-                                    width=2,
-                                ),
-                            ],
-                            align="center",
-                        ),
-                        dbc.Row(
-                            [
-                                dbc.Col(
-                                    [
-                                        dcc.DatePickerRange(
-                                            id=f"date-picker-range-{PAGE}-{VIZ_ID}",
-                                            min_date_allowed=dt.date(2005, 1, 1),
-                                            max_date_allowed=dt.date.today(),
-                                            initial_visible_month=dt.date(dt.date.today().year, 1, 1),
-                                            clearable=True,
-                                        ),
-                                    ],
+                                    width="auto",
                                 ),
                                 dbc.Col(
-                                    [
-                                        dbc.Button(
-                                            "About Graph",
-                                            id=f"popover-target-{PAGE}-{VIZ_ID}",
-                                            color="secondary",
-                                            size="sm",
-                                        ),
-                                    ],
```

Comment:
im not seeing where this about graph button got moved to within this visualization

Reply:
line 29